opencti-platform/opencti-graphql/src/initialization.js

```diff
@@ -291,6 +291,27 @@ export const createBasicRolesAndCapabilities = async (context) => {
     relationship_type: 'has-role',
   };
   await groupAddRelation(context, SYSTEM_USER, defaultGroup.id, defaultRoleRelationInput);
+  // Create connector group with connector role
+  const connectorGroup = await addGroup(context, SYSTEM_USER, {
+    name: 'Connectors',
+    description: 'Connector group',
+    auto_new_marking: true,
+  });
+  const connectorRoleRelationInput = {
+    toId: connectorRole.id,
+    relationship_type: 'has-role',
+  };
+  await groupAddRelation(context, SYSTEM_USER, connectorGroup.id, connectorRoleRelationInput);
+  // Markings
+  const markings = await findAll(context, SYSTEM_USER, { connectionFormat: false });
+  for (let i = 0; i < markings.length; i += 1) {
+    const marking = markings[i];
+    const connectorMarkingRelationInput = {
+      toId: marking.id,
+      relationship_type: 'accesses-to',
+    };
+    await groupAddRelation(context, SYSTEM_USER, connectorGroup.id, connectorMarkingRelationInput);
```

Comment:
I don't understand how this works, why do we need to add this relationship here since it's already done in `addMarkingDefinition` ?